debian/copyright

```diff
@@ -0,0 +1,41 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: eos-data-distribution
+
+Files: *
+Copyright:
+ © 2016 Endless Mobile, Inc.
+License: LGPL-3+
+
+Files:
+ dbus-service.py
```

Comment:
Indeed. We have a lot of code written in-house that copies the UCLA declaration. In fact, I don't think there's any UCLA-owned code in our repo at all.